samples/bookinfo/kube/istio-rbac-enable.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: "config.istio.io/v1alpha2"
+kind: authorization
+metadata:
+  name: requestcontext
+  namespace: istio-system
+spec:
+  subject:
+    user: source.user | ""
+    groups: source.user | ""
```

Comment:
I don't think it makes sense to repeat `source.user` for `groups`. Just leave it `""` since we don't use groups in this example.